Mathlib/RingTheory/Trace/Defs.lean

```diff
@@ -92,6 +92,10 @@ theorem trace_algebraMap_of_basis (b : Basis ι R S) (x : R) :
   convert Finset.sum_const x
   simp [-coe_lmul_eq_mul]
 
+/-- The trace map from `R` to itself is identity. -/
+@[simp] theorem trace_self (a : R) : trace R R a = a := by
+  simpa using trace_algebraMap_of_basis (.singleton (Fin 1) R) a
```

Comment:
Could you please prove the lemma about `trace R R` as well, without applying it to an element `a`?